web/cypress/integration/plugin/create-delete-in-drawer-plugin.spec.js

```diff
@@ -45,7 +45,6 @@ context('Create and Delete Plugin List', () => {
       .then(() => {
         cy.get('.ant-drawer-footer').contains('button', 'Delete').click();
         cy.contains('button', 'Confirm').click({ force: true });
-        cy.get(this.domSelector.empty).should('be.visible');
       });
```

Comment:
This assertion ensures that the plugin can be removed in its entirety properly, and I think it's best that we don't remove it

Reply:
any update here?